SpringMVCAnnotationOnlineStore/src/main/java/org/o7planning/springmvconlinestore/entity/Product.java

```diff
@@ -191,11 +160,11 @@ public String getColor() {
 	public void setColor(String color) {
 		this.color = color;
 	}
-	
+	/*
```

Comment:
@KendraMBach do you need a boolean `in-stock` in the db, or can you use `product.Number_In_Stock > 0`?

Reply:
Yep, I can just use a small utility function comparing `product.Number_In_Stock > 0` like you mentioned above. No need for an extra column from my perspective.